cookbooks/engineyard_docker/libraries/docker_service_manager_monit.rb

```diff
@@ -0,0 +1,51 @@
+module EngineyardDocker
+  class DockerServiceManagerMonit < ::DockerCookbook::DockerServiceBase
+    resource_name :docker_service_manager_monit
+
+    provides :docker_service_manager, platform: 'gentoo'
```

Comment:
we can probably look at dna.json to figure out if chef's running inside ey-gentoo.

But going with this for now is a good first pass.